src/libs/database.ts

```diff
@@ -0,0 +1,26 @@
+import * as mongoose from 'mongoose'
+import { promises, resolve } from 'dns';
+import { rejects } from 'assert';
```

Comment:
why are the packages imported? remove them

Reply:
Done